scripts/mina-local-network/mina-local-network.sh

```diff
@@ -552,6 +560,9 @@ printf "\n"
 # Start sending transactions
 
 if ${VALUE_TRANSFERS}; then
```

Comment:
We are separating these activities (regular value transfers and sending zkapp txns).
Please don't mix them in one block.

Reply:
I mixed those in a block because bash is single threaded and I need an infinite loop here to send both the zkApp transactions and normal transactions. I can clean this up a little bit though.

Reply:
Thanks